cs/rules.go

```diff
@@ -285,6 +287,7 @@ func NewRulesWithSeed(seed int64) Rules {
 			WormholeMinPlanetDistance: 30,
 		},
 		TachyonCloakReduction:            5,
+		TachyonMaxCloakReduction:         19, // cloak reduction caps at (100 - 19)% or 81%
```

Comment:
We should store this as a human readable value, so use 81, not 19. Don't make the player do math in their head if they are setting this as a custom rule.